internal/devices/assigned_devices.go

```diff
@@ -145,10 +145,7 @@ func readCsPipeOutput(l net.Listener, errChan chan<- error, result *[]string) {
 
 	elementsAsString := strings.TrimSuffix(string(bytes), "\n")
 	elements := strings.Split(elementsAsString, ",")
-
-	for _, elem := range elements {
-		*result = append(*result, elem)
-	}
+	*result = append(*result, elements...)
```

Comment:
haha did it really complain about this?

Reply:
Yes. There's a `gosimple` linter that tells you what can be simplified.